README.md

```diff
@@ -1275,10 +1275,10 @@ const app = express();
 const settings = require('db.json');
 const pool = require('node-querybuilder').QueryBuilder(settings, 'mysql', 'pool');
 
-app.post('/update_account', function(req, res) {
+app.post('/update_account', (req, res) => {
     const user_id = req.session.user_id;
-    const sanitize_name = function(name) { return name.replace(/[^A-Za-z0-9\s'-]+$/,'').trim(); };
-    const sanitize_age = function(age)  { return age.replace(/[^0-9]+$/,'').trim(); };
+    const sanitize_name = name => { return name.replace(/[^A-Za-z0-9\s'-]+$/,'').trim(); };
```

Comment:
Haha, I just realized the comment I made was for example code in the documentation. Oops. Well, if you wanna fix it, that's cool. If not, I'll just go ahead and merge, haha.